### DISCUSSION TITLE
marp cli server renders all markdown as slide

### DISCUSSION THREAD

**Discussion by jango-fx** [marp-team/marp#301] [Q&A]
As I understand it, the marp-cli server currently tries to display all Markdown files as slides.
This produces much unneeded clutter, if there are other .md files in the same folder.

In principle the YAML marker `marp: true` already exists and I would have expected that therefore only appropriately marked files would be rendered as slides.

Is there a command line option that causes a corresponding behavior and filters out non-marp files?

**Comment by yhatt** [comment]
There is no way for now. `marp: true` is just a dedicated directive to VS Code extension for enabling live preview and IntelliSense. Our tools treat all Markdown files equality.
